README.md

```diff
@@ -12,6 +12,35 @@ For more general-purpose Helm Charts, visit the [Helm Chart repository](https://
 
 Deis v2 and Helm are changing quickly. Your feedback and participation are more than welcome, but be aware that this project is considered a work in progress.
 
+## Contributing
+
+Hacking on Deis v2 is a bit rough right now--please help us fix the bugs and improve the installation process.
+
+First, add this Chart repo to Helm to install the "deis" chart:
+```console
+$ helm add repo deis https://github.com/deis/charts
+$ helm up
+$ helm fetch deis/deis
+$ helm install deis
+$ kubernetes get pods  # repeat this until deis-workflow is "Running"
+$ kubernetes get service deis-workflow  # note the IP address for later
```

Comment:
Ah thanks---I have it aliased to `k` so when I mentally expanded it there was apparently a cache miss. :-)